lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -330,6 +340,7 @@ def add_card(xml, payment_method, options)
         end
 
         xml.tag! 'cardHolderName', options[:execute_threed] ? '3D' : payment_method.name
+        # xml.tag! 'cardHolderName', payment_method.name #When testing 3DS 2.0 uncomment and comment above
```

Comment:
should we remove this comment?

Reply:
No we need to do this to test our remote tests from core unfortunately.

Reply:
Actually thought of a way to get rid of this.